pkg/cmd/server/api/validation/master.go

```diff
@@ -519,9 +519,9 @@ func ValidateRoutingConfig(config api.RoutingConfig) fielderrors.ValidationError
 }
 
 func ValidateAPIServerExtendedArguments(config api.ExtendedArguments) fielderrors.ValidationErrorList {
-	return ValidateExtendedArguments(config, kapp.NewAPIServer().AddFlags)
+	return nil //ValidateExtendedArguments(config, kapp.NewAPIServer().AddFlags)
```

Comment:
This makes me really unhappy. I think this is some of the most critical validation we have.  Is the cost just the size of the binary?